core/src/test/scala/unit/kafka/server/AddPartitionsToTxnRequestServerTest.scala

```diff
@@ -27,9 +27,9 @@ import org.junit.{Before, Test}
 
 import scala.jdk.CollectionConverters._
 
-class AddPartitionsToTxnRequestTest extends BaseRequestTest {
-  private val topic1 = "foobartopic"
-  val numPartitions = 3
+class AddPartitionsToTxnRequestServerTest extends BaseRequestTest {
```

Comment:
Why we want to change the test name?

Reply:
Because its name has a conflict with our added `AddPartitionsToTxnRequestTest`, just want to clarify that.